bin/running-sushi

```diff
@@ -315,17 +313,14 @@ def setup_config
     end
     opts.on('-v', '--verbosity', 'Verbosity level. Twice for debug.') do
       # If -vv is supplied this block is executed twice
-      if options[:verbosity]
-        options[:verbosity] = ::Logger::DEBUG
-      else
-        options[:verbosity] = ::Logger::INFO
-      end
+      options[:verbosity] = ::Logger::INFO
+      options[:verbosity] = ::Logger::DEBUG if options[:verbosity]
```

Comment:
Doesn't this unconditionally set `options[:verbosity] = ::Logger::DEBUG` ?
Due to the line above `options[:verbosity]` is always set to `::Logger::INFO` right?

Reply:
True, fixing.